nodeup/pkg/model/convenience.go

```diff
@@ -27,3 +34,31 @@ func s(v string) *string {
 func i64(v int64) *int64 {
 	return fi.Int64(v)
 }
+
+func getProxyEnvVars(proxies *kops.EgressProxySpec) []v1.EnvVar {
+	glog.Info("in getProxyEnvVars")
```

Comment:
Crap! I meant to remove those tracer logs, was troubleshooting something this weekend.  Thanks